### PR TITLE
fix(scripts): make /usr writable

### PR DESCRIPTION
**Pull request by therealbobo**
Signed-off-by: Roberto Scolaro <roberto.scolaro21@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> /kind release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

> /area tests

> /area proposals

> /area CI

**What this PR does / why we need it**:

`ProtectSystem` directive makes `/usr` directory read-only. By the way `falcoctl` needs write permission.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/systemd/falcoctl-artifact-follow.service [hunk 1]
```diff
@@ -13,7 +13,6 @@ RestartSec=15s
 Restart=on-failure
 PrivateTmp=true
 NoNewPrivileges=yes
-ProtectSystem=true
```
Comment (FedeDP) [inline scripts/systemd/falcoctl-artifact-follow.service:16]: Can we try `ReadWritePaths` instead?

Comment (FedeDP) [reply scripts/systemd/falcoctl-artifact-follow.service:16]: It looks like `ReadWritePaths` was introduced in systemd 231: https://github.com/systemd/systemd/blob/main/NEWS#L9059

I think we should use `ReadWriteDirectories` that is kept for backward compatibility:
> The old names of these settings remain
          available for compatibility.

Comment (FedeDP) [reply scripts/systemd/falcoctl-artifact-follow.service:16]: In falco-kmod.service too!

File: scripts/systemd/falco-kmod.service [hunk 2]
```diff
@@ -19,7 +19,7 @@ NoNewPrivileges=yes
 ProtectHome=read-only
 ProtectSystem=full
 ProtectKernelTunables=true
-ReadWritePaths=/sys/module/falco
+ReadWriteDirectories=/sys/module/falco
```
Comment (FedeDP) [inline scripts/systemd/falco-kmod.service:22]: Thanks!
@Andreagit97 we will need a quick test of the falco-kmod service after this change (from packages built by this pr ci)
This should work with super old versions of systemd as well (it was not working before 231).

**Review by FedeDP — APPROVED** [review]
Tested using CI packages. It works fine!
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**